src/Files.OpenDialog/FilesLauncher/OpenInFolder.cpp

```diff
@@ -73,37 +70,36 @@ LRESULT CALLBACK OpenInFolder::WindowProcedure(HWND hwnd, UINT Msg, WPARAM wPara
 	return DefWindowProc(hwnd, Msg, wParam, lParam);
 }
 
+
 HRESULT OpenInFolder::NotifyShellOfNavigation(PCIDLIST_ABSOLUTE pidl)
 {
 	wil::unique_variant pidlVariant;
 	RETURN_IF_FAILED(InitVariantFromBuffer(pidl, ILGetSize(pidl), &pidlVariant));
 
 	wil::unique_variant empty;
-
-	RETURN_IF_FAILED(m_shellWindows->RegisterPending(GetCurrentThreadId(), &pidlVariant, &empty,
-		SWC_BROWSER, &m_shellWindowCookie));
-
-	auto document = winrt::make_self<DocumentServiceProvider>();
-	auto shellView = winrt::make_self<ShellView>(this, pidl);
-	document->RegisterService(IID_IFolderView, shellView.get());
-
-	auto browserApp = winrt::make_self<WebBrowserApp>(m_hwnd, document.get());
-
-	long registeredCookie;
-#pragma warning(push)
-#pragma warning(                                                                                   \
-	disable : 4311 4302) // 'reinterpret_cast': pointer truncation from 'HWND' to 'long',
-	// 'reinterpret_cast': truncation from 'HWND' to 'long'
-	RETURN_IF_FAILED(m_shellWindows->Register(browserApp.get(), reinterpret_cast<long>(m_hwnd),
-		SWC_BROWSER, &registeredCookie));
-#pragma warning(pop)
+	RETURN_IF_FAILED(m_shellWindows->RegisterPending(GetCurrentThreadId(), &pidlVariant, &empty, SWC_BROWSER, &m_shellWindowCookie));
+
+//	auto document = winrt::make_self<DocumentServiceProvider>();
+//	auto shellView = winrt::make_self<ShellView>(this, pidl);
+//	document->RegisterService(IID_IFolderView, shellView.get());
+//	auto browserApp = winrt::make_self<WebBrowserApp>(m_hwnd, document.get());
+//	long registeredCookie;
+//
+//// Supress:
+////  - 'reinterpret_cast': pointer truncation from 'HWND' to 'long',
+////  - 'reinterpret_cast': truncation from 'HWND' to 'long'
+//#pragma warning(push)
+//#pragma warning(disable : 4311 4302)
+//	RETURN_IF_FAILED(m_shellWindows->Register(browserApp.get(), reinterpret_cast<long>(m_hwnd), SWC_BROWSER, &registeredCookie));
+//#pragma warning(pop)
```

Comment:
Delete the commented out code.

Reply:
Finally, yes. still remain.

Reply:
Probably shouldn't remove it in order to revert this only if the critical issue appears. But I'm not sure.

Reply:
I don't think it is necessary to leave the commented out code because deleted code can be easily restored from the commit history.